### PR TITLE
BTR-3032 | Add support for set image sizes in order to improve cache hit ratio

### PR DESCRIPTION
**Pull request by tomrains**
Adding in preset image widths in order as part of our initiative to improve the cache hit ratio. Relevant to https://hiburrow.atlassian.net/browse/BTR-2690

Now, instead of dynamically recalculating what image widths to fetch every time browser is resized, we fetch some basic ones and let the browser handle the rest. This means fewer calls for these images when later visiting the site, and relying more on cached images

I chose the asset widths based on the suggestions from this article: https://medium.com/hceverything/applying-srcset-choosing-the-right-sizes-for-responsive-images-at-different-breakpoints-a0433450a4a3

### CODE REVIEW

File: test/index.test.js [hunk 1]
```diff
@@ -1,24 +1,29 @@
...
-
   test('should construct correct srcSet', async () => {
+    const presetImageSizes = [
+      640,
+      768,
+      1024,
+      1366,
+      1600,
+      1920,
+    ]
```
Comment (giankotarola) [inline test/index.test.js]: i think we can export `presetImageSizes` from `src/index.js` so we can re use it here

**Review by giankotarola — CHANGES_REQUESTED** [review]
looks good! just a note in order to reused the `presetImageSizes` variable from `src/index.js`

File: test/index.test.js (hunk 1 above)
Comment (tomrains) [reply test/index.test.js]: thanks! fixed in https://github.com/hiburrow/graphcms-image/pull/3/commits/1f608c94e498deec278e0bf355ba85c8a93a884f

**Review by giankotarola — APPROVED** [review]
LGTM!

**Review by rafasirotheau — APPROVED** [review]

**Review by giankotarola — APPROVED** [review]

**Pull request merged**